fetch.bs

```diff
@@ -4503,12 +4503,14 @@ steps:
     <a for="fetch params">canceled</a>:
 
     <ol>
-     <li><p>Let <var>blob</var> be <var>request</var>'s <a for=request>current URL</a>'s
-     <a for=url>blob URL entry</a>'s <a for="blob URL entry">object</a>.
+     <li><p>Let <var>blobURLEntry</var> be the associated <a for=url>blob URL entry</a> of
```

Comment:
Much of the rewording here may not be necessary. My intent was to avoid using successive possessive nouns, which could improve readability. I'm certainly open to suggestions here.

Reply:
We tend to use the current style in specifications so I'd suggest we stick to that as it might actually hurt readability overall when you keep switching styles.

Reply:
:+1: Sounds good